net/libndp/Makefile

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (C) 2017 OpenWrt.org
```

Comment:
General suggestion, don't assign copyright to OpenWrt.org unless you're affiliated.
https://forum.lede-project.org/t/packaging-guidelines/65

Reply:
Ok I see that.... I don't fully agree about the principle though, it's easier to manage licencing changes to a project when it owns copyright... will fix.